bundles/org.openhab.binding.gardena/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -10,1162 +10,633 @@
-		<description>The source for the next start</description>
-		<state readOnly="true">
-			<options>
-				<option value="no_source">No source</option>
-				<option value="completed_cutting_daily_limit">Completed cutting daily limit</option>
-				<option value="week_timer">Week timer</option>
-				<option value="countdown_timer">Countdown timer</option>
-				<option value="mower_charging">Mower charging</option>
-				<option value="completed_cutting_autotimer">Completed cutting autotimer</option>
-				<option value="undefined">Undefined</option>
-			</options>
-		</state>
-	</channel-type>
+	<!-- ############################### Sensor ############################### -->
 
-	<channel-type id="timestampNextStart">
-		<item-type>DateTime</item-type>
-		<label>Timestamp Next Start</label>
-		<description>Timestamp of the next start</description>
-		<state readOnly="true"/>
-	</channel-type>
+	<thing-type id="sensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="account"/>
+		</supported-bridge-type-refs>
 
-	<channel-type id="overrideEndTime" advanced="true">
-		<item-type>DateTime</item-type>
-		<label>Override End Time</label>
-		<description>Override End Time</description>
-		<state readOnly="true"/>
-	</channel-type>
+		<label>Gardena smart Sensor</label>
+		<description>Represents a Gardena smart Sensor</description>
 
-	<channel-type id="temperature" advanced="true">
-		<item-type>Number</item-type>
-		<label>Temperature</label>
-		<description>Temperature</description>
-		<state pattern="%d °C" readOnly="true"/>
-	</channel-type>
+		<channel-groups>
+			<channel-group id="common" typeId="commonProperties"/>
+			<channel-group id="sensor" typeId="sensorProperties"/>
+		</channel-groups>
+		<properties>
+			<!-- common -->
+			<property name="serial"/>
+			<property name="modelType"/>
+		</properties>
+	</thing-type>
 
-	<channel-group-type id="firmware">
-		<label>Firmware</label>
-		<description>Information about the firmware</description>
-		<channels>
-			<channel id="firmware_status" typeId="firmwareStatus"/>
-			<channel id="firmware_upload_progress" typeId="firmwareUploadProgress"/>
-			<channel id="firmware_available_version" typeId="firmwareAvailableVersion"/>
-		</channels>
-	</channel-group-type>
+	<!-- ############################### Pressure Pump ############################### -->
 
-	<channel-type id="firmwareStatus" advanced="true">
-		<item-type>String</item-type>
-		<label>Status</label>
-		<description>The firmware status</description>
-		<state readOnly="true"/>
-	</channel-type>
+	<thing-type id="pump">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="account"/>
+		</supported-bridge-type-refs>
 
-	<channel-type id="firmwareUploadProgress" advanced="true">
-		<item-type>Number</item-type>
-		<label>Upload Progress</label>
-		<description>Firmware upload in progress</description>
-		<state readOnly="true"/>
-	</channel-type>
+		<label>Gardena smart Pressure Pump</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions